src/components/level-field/level-field-component.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  BaseHtmlOptions,
+  BaseHTML
+} from '@components/baseHTML/BaseHTML';
+import {
+  BAR_EMPTY,
+  BAR_FULL,
+  LEVEL_INDICATOR
+} from "@constants";
+import './level-field-component.scss';
+
+export const DEFAULT_SELECTORS = {
+  root: '.game-scene', //Class name of game scene div.
+  bars: '.bar-level'
+};
+
+
+export const LEVEL_FIELD_LAYOUT = (id: string) => {
+  return (`
+    <div id="${id}" class="level_content-wrapper">
+      <div class="level-field">
+        <img class="level-background" src="${LEVEL_INDICATOR}"/>
+        <div class="bar-level-wrapper">
+          <img class="bar-level" src="${BAR_EMPTY}" />
```

Comment:
I have modified this to more optimized way also add ALT property

<div class="bar-level-wrapper">
  ${Array(5).fill(`<img class="bar-level" src="${BAR_EMPTY}" alt="any content/>`).join('')}
  </div>